### ISSUE TITLE
Wadanohara: Slow movement when fighting Sal

### ISSUE THREAD

**Issue by Ghabry** [EasyRPG/Player#1351]
This is already reported soo often via Mail that it needs further investigation.

Not sure yet if 0.5.3 only and already fixed or not.

[Save.zip](https://github.com/EasyRPG/Player/files/1967724/Save.zip)

Bug report:
"WHY IS WADANOHARA IS SO FREAKING SLOW THAT I ALWAYS GET KILLED OR RAPED BY SAL!!! PLEASE FIX MY WADANOHARA AND HOPEFULLY TOMORROW WADANOHARA CAN WALK OR RUN FASTER NOW NOT LIKE WALKING IR RUNNING LIKE A SNAIL" thanks you that is my report and PLEASE FIX WADANOHARA!"

"Wadanohara no avanza de forma normal, se mueve demaciado lento." (Wadanohara doesn't advance normal, she moves very slow)

**Comment by vgperson** [comment]
Well, not sure if this helps, but I did a playthrough in the latest EasyRPG build (617) all the way up to the Sal chase part, and didn't seem to encounter any speed bugs.

[WadanoharaTestSaves.zip](https://github.com/EasyRPG/Player/files/1978475/WadanoharaTestSaves.zip)

Here are some periodic saves I made during the playthrough. Save04 is from the last time you have control before the submitted Save01. Save06 is around the same area as the submitted Save02. And Save09 is before the Sal chase, while Save10 is during it.
